src/gui/torrentcategorydialog.cpp

```diff
@@ -29,6 +29,7 @@
 #include "torrentcategorydialog.h"
 
 #include <QMessageBox>
+#include <qdialogbuttonbox.h>
```

Comment:
`#include <QDialogButtonBox>`.

Reply:
And this header order is wrong.

Reply:
My bad. `#include` is automatically added when I just use autocomplete feature. This shouldn't be here.